azure/route.sh

```diff
@@ -12,6 +12,34 @@ fn azure_route_table_delete(name, group) {
 	)
 }
 
+fn azure_route_table_route_new(name, group, routetable, address, hoptype) {
+	instance = (
+		"--name"
+		$name
+		"--resource-group"
+		$group
+		"--route-table-name"
+		$routetable
+	    "--address-prefix"
+		$address
+	    "--next-hop-type"
+		$hoptype
+	)
+
+	return $instance
+}
+
+fn azure_route_table_route_set_hop_address(instance, hopaddress) {
```

Comment:
The "--next-hop-ip-address" option is valid only for next hop type VirtualAppliance.